templates/_http.conf.erb

```diff
@@ -1 +1,2 @@
 ProxyPass /unattended http://localhost:3000/unattended
+Redirect permanent <%= scope['katello_devel::deployment_url'] %>/api https://<%= scope['katello_devel::fqdn'] %><%= scope['katello_devel::deployment_url'] %>/api
```

Comment:
As far as I can tell we don't need to. I updated the redirect to redirect / to https://fqdn/